packages/kotti-ui/source/kotti-tag/types.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from 'zod'
+
+export namespace KottiTag {
+	export const propsSchema = z.object({
```

Comment:
I propose an `isDisabled` that prohibits the interaction all together - would be needed to sustain old behavior on the GenericSelectField as well